css/main.css

```diff
@@ -12,7 +12,7 @@
 @font-face {
   font-family: NeuzeitGroT;
   /* Chrome 4+, Firefox 3.5, Opera 10+, Safari 3—5 */
-  src: url('../resources/fonts/NeuzeitGroT-2.ttf') format('truetype');
+  src: url('../../resources/fonts/NeuzeitGroT-2.ttf') format('truetype');
```

Comment:
Shouldn't this be `src: url('../resources/fonts/NeuzeitGroT-2.ttf') format('truetype');` ? Since the root folder is 1 folder back from the css/main.js.

Reply:
We forgot to remove the '../' when we moved these lines of CSS from profile/main.css to main.css